### PR TITLE
ufw: add page

### PR DESCRIPTION
**Pull request by Torild**

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks for your contribution! A suggestion: how about adding [the meaning of ufw](https://en.wikipedia.org/wiki/Uncomplicated_Firewall) in the main description? That would help people memorize the command.

**Comment by Torild** [comment]
Sure thing. I changed the description a bit

File: pages/linux/ufw.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## ufw
+
+> Ufw (Uncomplicated Firewall) is a frontend for iptables aiming to make configuration of a firewall easier
```
Comment (igorshubovych) [inline pages/linux/ufw.md]: Can be simplified

```markdown
> Uncomplicated Firewall
> frontend for iptables aiming to make configuration of a firewall easier
```

File: pages/linux/ufw.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`ufw enable`
+
+- Disable ufw
+
+`ufw disable`
+
+- Add ufw allow rule
+
+`ufw allow {{port or service name}}`
```
Comment (igorshubovych) [inline pages/linux/ufw.md]: Maybe split {{port or service name}}` into 2 examples.

**Comment by waldyrious** [comment]
Are the last two examples technically correct? The first version of this PR suggested that the parameter could be either a port or a service name, not both. The new version suggests both can be passed. So what's the correct invocation?

**Comment by Torild** [comment]
It can be either, not both

**Comment by G570972680893** [comment]
А люди где?
2 янв. 2016 г. 16:30 пользователь "Torild" <notifications@github.com>
написал:

**Comment by waldyrious** [comment]
In that case I suppose the example should be split it two to prevent confusion.

File: pages/linux/ufw.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`ufw disable`
+
+- Add ufw allow rule
+
+`ufw allow {{port}} {{service name}}`
+
+- Add ufw deny rule
+
+`ufw deny {{port}} {{service name}}`
```
Comment (igorshubovych) [inline pages/linux/ufw.md]: We earlier agreed to have argument names without spaces, so it should be
{{service_name}}

**Comment by igorshubovych** [comment]
Check out linter. It says:

pages/linux/ufw.md:1: MD002 First header should be a h1 header
pages/linux/ufw.md:3: MD009 Trailing spaces

**Comment by igorshubovych** [comment]
Also please squash commits.

**Comment by waldyrious** [comment]
Don't forget to split the last two examples to make sure people don't get the impression they can pass both parameters simultaneously.

**Comment by igorshubovych** [comment]
@Torild the page already looks good.
Could you please squash these 6 commits into one?
http://take.ms/uihhF

Here is the instruction on how to do it:
https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request

**Comment by Torild** [comment]
I squashed the commits and pushed them to my fork, but I'm not quite sure how to get them to show up here?

**Comment by igorshubovych** [comment]
You have to do 

`git push --force`.

**IMPORTANT**
But be careful, check log (`git log`) of your branch, it should contain only one commit on top of checkout commit (since you did `git checkout -b`)

**Comment by rubenvereecken** [comment]
Good stuff, thanks!

**Comment by waldyrious** [comment]
@rubenvereecken how do we address the `{{port}}` vs `{{service_name}}` issue raised above, without exceeding the line count / max. number of examples? Use one for allow and another for deny? Would that be clear enough?

**Comment by rubenvereecken** [comment]
You're right for raising the issue again @waldyrious, annoying I merged that in. We can do like you suggest, it's a bit confusing but it'll work. Too bad we don't use concrete examples anymore, otherwise we could have used "80" and "http" or something, well-known ports that make it easy to grasp for anyone.

**Comment by waldyrious** [comment]
Perhaps if the descriptions mention the extra option, people can infer from the similarity how it can be composed otherwise.

**Pull request merged**